stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -56,18 +56,16 @@ spec:
          - |
             #persist the erlang cookie in both places for server and cli tools
             echo $RABBITMQ_ERL_COOKIE > /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie
-            cp /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie /root
-
+            cp /opt/bitnami/rabbitmq/var/lib/rabbitmq/.erlang.cookie /opt/bitnami/rabbitmq/.rabbitmq/
             #change permission so only the user has access to the cookie file
-            chmod 400 /root/.erlang.cookie
+            chmod 400 /opt/bitnami/rabbitmq/.rabbitmq/.erlang.cookie
```

Comment:
it it not necessary to specify that the erlang cookie is in a new path?

Reply:
No, because $HOME is set as /opt/bitnami/rabbitmq/.rabbitmq/